docs/source/examples/capacity_expansion_tutorial.ipynb

```diff
@@ -54,9 +54,11 @@
     "\n",
     "# set the solver based on operating system -- assumes glpk or cbc is installed.\n",
     "if \"win32\" in sys.platform:\n",
-    "    solver = 'cplex'\n",
+    "    solver = 'cplex_direct'\n",
```

Comment:
How do you know this line needs to be changed? Have you tested this on a windows computer? (I have and it seems to work fine).